src/Routers/SchemasRouter.js

```diff
@@ -35,7 +35,42 @@ function getOneSchema(req) {
     });
 }
 
-function createSchema(req) {
+const checkIfDefinedSchemasIsUsed = req => {
+  if (req.config && req.config.schema && req.config.schema.lockSchemas === true) {
+    throw new Parse.Error(
+      Parse.Error.OPERATION_FORBIDDEN,
+      'Cannot perform this operation when schemas options is used.'
+    );
+  }
+};
+
+export const internalCreateSchema = async (className, body, config) => {
+  const controller = await config.database.loadSchema({ clearCache: true });
+  return {
+    response: await controller.addClassIfNotExists(
```

Comment:
personally not a fan of using `await` in objects, I would rather see

```js
const controller = await ...
const response = await ...
return {  response  }
```

Reply:
on it